test/mocha_integration.test.js

```diff
@@ -12,3850 +14,3982 @@ const itHelpers = require("./integration_test_misc/integration_test_helpers");
-    );
-    resBody = JSON.parse(res.body.toString("utf8"));
-    expect(resBody).to.deep.equal({
-      data: {
-        sq_booksConnection: {
-          edges: [
-            { node: { id: "local_b1", author_ids: [] } },
-            { node: { id: "local_b2", author_ids: [] } },
-            { node: { id: "remote_b1", author_ids: [] } },
-            { node: { id: "remote_b2", author_ids: [] } },
-          ],
-        },
-      },
-    });
-  });
-
-  it("10. Create record and add association - ddm sql adapter", function () {
-    let res = itHelpers.request_graph_ql_post_instance2(
-      'mutation{addSq_author(id:"local_a1" name:"n1" addBooks:["local_b1","local_b2"]){id book_ids}}'
-    );
-    expect(res.statusCode).to.equal(200);
-    let resBody = JSON.parse(res.body.toString("utf8"));
-    //check it has been created correctly
-    expect(resBody.data).to.deep.equal({
-      addSq_author: { id: "local_a1", book_ids: ["local_b1", "local_b2"] },
-    });
-    //check inverse association
-    res = itHelpers.request_graph_ql_post_instance2(
-      "{sq_booksConnection(pagination:{first: 4}){ edges { node {id author_ids} }}} "
-    );
-    resBody = JSON.parse(res.body.toString("utf8"));
-    expect(resBody.data).to.deep.equal({
-      sq_booksConnection: {
-        edges: [
-          { node: { id: "local_b1", author_ids: ["local_a1"] } },
-          { node: { id: "local_b2", author_ids: ["local_a1"] } },
-          { node: { id: "remote_b1", author_ids: [] } },
-          { node: { id: "remote_b2", author_ids: [] } },
-        ],
-      },
-    });
-  });
-
-  it("11. Query associated records - ddm sql adapter", function () {
-    let res = itHelpers.request_graph_ql_post_instance2(
-      '{readOneSq_book(id:"local_b1"){id authorsConnection(pagination: {first: 2}){ edges{ node{id}}}}}'
-    );
-    expect(res.statusCode).to.equal(200);
-    let resBody = JSON.parse(res.body.toString("utf8"));
-    //check associated records
-    expect(resBody.data).to.deep.equal({
-      readOneSq_book: {
-        id: "local_b1",
-        authorsConnection: { edges: [{ node: { id: "local_a1" } }] },
-      },
-    });
-  });
-
-  it("12. Update record and remove association - ddm sql adapter", function () {
-    let res = itHelpers.request_graph_ql_post_instance2(
-      'mutation{updateSq_author(id:"local_a1" name:"n1" removeBooks:["local_b1","local_b2"]){id book_ids}}'
-    );
-    expect(res.statusCode).to.equal(200);
-    let resBody = JSON.parse(res.body.toString("utf8"));
-    //check it has been updated correctly
-    expect(resBody.data).to.deep.equal({
-      updateSq_author: { id: "local_a1", book_ids: [] },
-    });
-    //check inverse association
-    res = itHelpers.request_graph_ql_post_instance2(
-      "{sq_booksConnection(pagination:{first: 4}){ edges { node {id author_ids} }}}"
-    );
-    resBody = JSON.parse(res.body.toString("utf8"));
-    expect(resBody).to.deep.equal({
-      data: {
```

Comment:
seems like the indent is not consistent. Previously we use indent with 2 characters. Currently the indent has four characters. Maybe adjust the prettier setup?